src/identity/checkHashLength.ts

```diff
@@ -0,0 +1,3 @@
+export function checkHashLength(hash: string, length: number): hash is `0x${string}` {
```

Comment:
Can you add JS docs on this please.

Also should this maybe live in `src/utils` ?